src/store/modules/wallet.js

```diff
@@ -30,6 +35,7 @@ const state = () => ({
   loginAddress: '',
   connector: null,
   likerInfo: null,
+  followers: null,
```

Comment:
Empty array is better
```suggestion
  followers: [],
```